src/pages/Login.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <ion-page>
+     <Toolbar title-text="Login"/>
+
+    <ion-content :fullscreen="true">
+      <ion-form @submit.prevent="handleSubmit">
+        <div id="login">
+          <ion-item>
+            <ion-input
+              placeholder="Email"
+              type="email"
+              ion-model="email"
+            ></ion-input>
+          </ion-item>
+          <ion-item>
+            <ion-input
+              placeholder="Password"
+              type="password"
+              ion-model="password"
+            ></ion-input>
+          </ion-item>
+          <ion-button type="submit">Login</ion-button>
+          <ion-button @click="$router.push('/register')">Register</ion-button>
+        </div>
+      </ion-form>
+    </ion-content>
+  </ion-page>
+</template>
+
+<script lang="ts">
+import {
+  IonContent,
+  IonInput,
+  IonPage
+} from "@ionic/vue";
+import { defineComponent } from "vue";
+import { useRouter } from "vue-router";
+import axios from "axios";
+import Toolbar from '@/components/Toolbar.vue'
+
+export default defineComponent({
+  name: "Home",
+  components: {
+    IonContent,
+    IonPage,
+    IonInput,
+    Toolbar
+  },
+  setup() {
+    const router = useRouter();
+    return { router };
+  },
+  data() {
+    return {
+      email: "",
+      password: "",
+    };
+  },
+  methods: {
+    async handleSubmit() {
+      const response = await axios.post("login", {
+        email: this.email,
+        password: this.password,
+      });
```

Comment:
Create a user service to handle this kind of stuff and just inject them anywhere in the app. In the future you won't just handle the login, but the registration, updating of profile, passwords, etc. We want to contain them in a single instance and just reuse them. It will also be easier to refactor them or there's a breaking change since there will be just one file affected.